Cartfile

```diff
@@ -1,5 +1,5 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.5
-github "mapbox/MapboxDirections.swift" ~> 0.10.0
+github "mapbox/MapboxDirections.swift" ~> 0.10
```

Comment:
Since MapboxDirections.swift hasn’t yet reached v1.0, anything could technically change at any time. We do try to avoid backwards-compatible changes in patch releases, but we make no such guarantees about minor releases.

We should pin to a specific patch version of the library (with the tadpole operator) to avoid surprises. `~> 0.10.1` will require v0.10.x except v0.10.0.